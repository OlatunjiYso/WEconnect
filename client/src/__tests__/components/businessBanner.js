import React from "react";

import BusinessBanner from '../../components/businessBanner';

describe('All tests for business banner component', () => {
 
  const props = {
    business: {
      name: '',
      image: ''
    }
  };
  
    it('should be defined', () => {
      expect(BusinessBanner).toBeDefined();
    });
    it('should render the banner', () => {
      const { wrapper } = shallow(<BusinessBanner {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
});
