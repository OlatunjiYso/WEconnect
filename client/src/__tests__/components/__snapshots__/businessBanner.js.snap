// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for business banner component should render the banner 1`] = `undefined`;
